src/pe_2.py

```diff
@@ -6,16 +6,12 @@
 
 By considering the terms in the Fibonacci sequence whose values do not exceed four million, find the sum of the even-valued terms.'''
 
-a,b=[],[]
-c,d=0,1
+total = 0
+a,b = 0, 1
 
-while c<4000000:
-  a.append(c)
-  c,d=d,c+d
-  
-for num in a[1:len(a)]:
-  if num % 2 == 0:
-    b.append(num)
-    continue
-    
-sum(b) #4613732
+while a<4000000:
+  a, b = b, a + b
+  if a % 2 == 0:
+    total = total + a
```

Comment:
Python's syntax for updating the value of a variable can be a little more succinct than in R:

```python
    total += a
```